ISSUE_TEMPLATE.md

```diff
@@ -15,6 +15,9 @@ Further details:
   - [ ] I have read and filled out the [Self-Review Questionnare on Security and Privacy](https://www.w3.org/TR/security-privacy-questionnaire/). The [assessment is here](url).
   - [ ] I have reviewed the TAG's [API Design Principles](https://w3ctag.github.io/design-principles/)
   - The group where the work on this specification is:
+  
+We recommend the explainer to be in [Markdown](https://github.github.com/gfm/). On top of the [usual information expected in the explainer](https://w3ctag.github.io/explainers), it is strongly recommended to add:
```

Comment:
> We recommend the explainer to be in [Markdown](https://github.github.com/gfm/)

Was this in response to comments [similar to mine](We recommend the explainer to be in [Markdown](https://github.github.com/gfm/))? If so, thank you very much for this. :blush: